public_custom_application_environments/python39_streamlit/Dockerfile

```diff
@@ -9,8 +9,10 @@ RUN apt-get update && apt-get install -y \
     git \
     && rm -rf /var/lib/apt/lists/*
 
+COPY ./packages/dr_components-0.1.2-py3-none-any.whl /app/
```

Comment:
It looks like you haven't added `packages` folder to repo.

Reply:
oh, thanks for catching! :face_in_clouds:  Fixed now